docs/source/user/aerodyn/examples/ad_primary_example.dat

```diff
@@ -19,26 +17,36 @@ False         CompAA             - Flag to compute AeroAcoustics calculation [us
-"default"     SkewModFactor      - Constant used in Pitt/Peters skewed wake model {or "default" is 15/32*pi} (-) [used only when SkewMod=2; unused when WakeMod=0 or 3]
-f             TipLoss            - Use the Prandtl tip-loss model? (flag) [unused when WakeMod=0 or 3]
-f             HubLoss            - Use the Prandtl hub-loss model? (flag) [unused when WakeMod=0 or 3]
+======  Blade-Element/Momentum Theory Options  ====================================================== [unused when WakeMod=0 or 3, except for BEM_Mod]
+          2   BEM_Mod            - BEM model {1=legacy NoSweepPitchTwist, 2=polar} (switch) [used for all Wake_Mod to determine output coordinate system]
+--- Skew correction
+          1   Skew_Mod           - Skew model {0=No skew model, -1=Remove non-normal component for linearization, 1=skew model active}
+True          SkewMomCorr        - Turn the skew momentum correction on or off [used only when Skew_Mod=1]
+          1   SkewRedistr_Mod    - Type of skewed-wake correction model (switch) {0=no redistribution, 1=Glauert/Pitt/Peters} [used only when Skew_Mod=1]
+"default"     SkewRedistrFactor  - Constant used in Pitt/Peters skewed wake model {or "default" is 15/32*pi} (-) [used only when Skew_Mod=1 and SkewRedistr_Mod=1]
+--- BEM algorithm 
+True          TipLoss            - Use the Prandtl tip-loss model? (flag) [unused when WakeMod=0 or 3]
+True          HubLoss            - Use the Prandtl hub-loss model? (flag) [unused when WakeMod=0 or 3]
 True          TanInd             - Include tangential induction in BEMT calculations? (flag) [unused when WakeMod=0 or 3]
 True          AIDrag             - Include the drag term in the axial-induction calculation? (flag) [unused when WakeMod=0 or 3]
 True          TIDrag             - Include the drag term in the tangential-induction calculation? (flag) [unused when WakeMod=0,3 or TanInd=FALSE]
-      1E-05   IndToler           - Convergence tolerance for BEMT nonlinear solve residual equation {or "default"} (-) [unused when WakeMod=0 or 3]
+"Default"     IndToler           - Convergence tolerance for BEMT nonlinear solve residual equation {or "default"} (-) [unused when WakeMod=0 or 3]
         100   MaxIter            - Maximum number of iteration steps (-) [unused when WakeMod=0]
-======  Dynamic Blade-Element/Momentum Theory Options  ============================================== [used only when WakeMod=2]
-          2   DBEMT_Mod          - Type of dynamic BEMT (DBEMT) model {1=constant tau1, 2=time-dependent tau1, 3=constant tau1 with continuous formulation} (-) [used only when WakeMod=2]
-          4   tau1_const         - Time constant for DBEMT (s) [used only when WakeMod=2 and DBEMT_Mod=1 or 3]
+--- Shear correction 
+False         SectAvg           - Use sector averaging (flag)
+1             SectAvgWeighting  - Weighting function for sector average  {1=Uniform}  within a sector centered on the blade (switch) [used only when SectAvg=True] 
```

Comment:
Were other SectAvgWeighting options implemented?  Wasn't the angle range of the sector averaging implemented?  Or do we want to keep this simple for now?

Reply:
Currently I have two hidden inputs: 
`SA_PsiBwd` and `SA_PsiFwd` 
We disagreed on their use (you advocated for 360/nB) and wanted to rely on the weighting method only. 
I think the extent is quite useful, and saves in computational time (if we want to focus close to the blade). 
I would be in favor of introducing these inputs below. The alternative is to keep them hidden for now.

Reply:
I generally prefer not having "hidden" inputs.  I would be in favor of exposing the inputs you recommend (`SA_PsiBwd`, `SA_PsiFwd`) directly within the input file.